### ISSUE TITLE
Getting weird formatting for my dates

### ISSUE THREAD

**Issue by josiahcoad** [stephane-monnot/react-vertical-timeline#19]
![image](https://user-images.githubusercontent.com/15700085/45590965-b3135400-b90a-11e8-923f-3db26fd4f922.png)

**Comment by Jeyloh** [comment]
Did you use * { box-sizing: border-box } ? It will remove the padding/margin on datestring which puts it in the correct position.

**Comment by stephane-monnot** [comment]
I think the problem is likely due to box-sizing: border-box

Maybe related to this issue : #3

**Comment by goughjo02** [comment]
hello. I encountered this issue as well. I am glad to find this thread. 

I am using material-ui-react and I am using the `CssBaseline` component. (https://material-ui.com/style/css-baseline/)

afaik this component applies `box-sizing: border-box` by default, as such, I had to edit the minified css file to fix the positioning of the dates. I notice that the VerticalTimelineElement is also slightly off the vertical center. This seems like an issue. I would love a prop to deal with this. 

Thanks for the awesome package. Honestly, well done.
